packages/metrics/src/features/registration/index.ts

```diff
@@ -57,6 +66,11 @@ export interface IBirthRegistrationTags {
   locationLevel2?: string
 }
 
+export interface IUserAuditTags {
+  action: DECLARATION_STATUS | null | string
```

Comment:
Can action be null? I think the type `USER_ACTION` you've defined would be pretty good here

```ts
// ...
action: USER_ACTION
```